### PR TITLE
rewrite the quickstart guide

### PR DESCRIPTION
**Pull request by HarshCasper**
https://localstack-docs-preview-pr-976.surge.sh/getting-started/quickstart/

### CODE REVIEW

**Comment by MarcelStranak** [comment]
Hi @HarshCasper, I tried to follow the steps, but I received this error: 

```
pip install -r requirements-dev.txt  

 RuntimeWarning: Pillow 9.2.0 does not support Python 3.12 and does not provide prebuilt Windows binaries. We do not recommend building from source on Windows.
```

Prerequisites mention **Python 3.9+**

For now, I will follow the guide with a newer version of the pillow plugin.

```
Pillow-10.1.0-cp312-cp312-win_amd64.whl
```

File: content/en/getting-started/quickstart/index.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+---
+
+In this quickstart guide, we'll walk you through the process of starting LocalStack on your local machine and deploying a [serverless image resizer application](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda) that utilizes several AWS services. This guide aims to help you understand how to use LocalStack for the development and testing of your AWS applications locally. It introduces you to the following key concepts:
+
+- Starting a LocalStack instance on your local machine.
+- Deploying an AWS serverless application infrastructure locally.
+- Running an automated integration test suite against local infrastructure.
+- Exploring the LocalStack Web Application to view deployed resources.
+- Destroying the local infrastructure you have provisioned.
+
```
Comment (SimonWallner) [inline content/en/getting-started/quickstart/index.md:18]: add a `#The example App` Section that briefly describes what we are building, ideally with screenshots or animated gif showing it in action.

note: I've noticed that we use the pikachu in the below screenshots. This is quite certainly a copy right violation, and we should only use images that are our own or are in public domain.

File: content/en/getting-started/quickstart/resized-image-sample-application.png
Comment (SimonWallner) [inline content/en/getting-started/quickstart/resized-image-sample-application.png:1]: copyrighted material

**Review by SimonWallner — APPROVED** [review]
LGTM, looked mostly at the structure, not the techincal details. See comments

File: content/en/getting-started/quickstart/index.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+cd libs && zip -r ../lambda.zip . && cd ..
+zip lambda.zip handler.py
+rm -rf libs
+cd ../..
+{{< /tab >}}
+{{< tab header="Linux/Windows" lang="shell" >}}
+cd lambdas/resize
+rm -rf package lambda.zip
+mkdir package
+pip install -r requirements.txt -t package
```
Comment (MarcelStranak) [inline content/en/getting-started/quickstart/index.md:201]:
```suggestion
pip install -r requirements.txt -t package --platform manylinux_2_28_x86_64 --python-version 3.9 --no-deps
```

This needs to be changed in the requirements.txt or in the command. As you are utilizing Linux lambda to execute it, it is necessary that you specify the platform and version to align with it. If you do not, pip will download Windows version of the python package.

**Comment by MarcelStranak** [comment]
@HarshCasper I propose that the Windows commands will be removed for the moment and added again after. 
Going through it is a bit of a hassle. I can help you with the PowerShell script as well.

**Review by tinyg210 — APPROVED** [review]
LGTM; i'm not familiar with the power shell part, so i would take Marcel's advice and remove it or take his help with it

**Comment by thrau** [comment]
@HarshCasper can you incorporate the feedback so we can get it over the line?

**Comment by lakkeger** [comment]
In the requirements it might worth to mention pip install too as we've got feedback from non-python developers that they have no idea what pip is.

**Review by thrau — APPROVED** [review]
great changes! :100: 

maybe in a second iteration we can get rid of the local python dependency completely, at least for basic development.

**Pull request merged**